notebooks/2016-12-19-exploring_csw.ipynb

```diff
@@ -2,27 +2,23 @@
  "cells": [
   {
    "cell_type": "markdown",
-   "metadata": {
-    "collapsed": false,
-    "deletable": true,
-    "editable": true
-   },
+   "metadata": {},
    "source": [
     "# How to search the IOOS CSW catalog with Python tools\n",
     "\n",
     "\n",
-    "This notebook demonstrates a how to query a [Catalog Service for the Web (CSW)](https://en.wikipedia.org/wiki/Catalog_Service_for_the_Web), like the IOOS Catalog, and to parse its results into endpoints that can be used to access the data.\n",
-    "\n",
-    "(This notebook uses a custom `ioos_tools` module that needs to be added to the path separately. We recommend cloning the [repository](https://github.com/ioos/notebooks_demos) on GitHub which already includes the most update version of `ioos_tools`.)"
+    "This notebook demonstrates a how to query the IOOS Catalog [Catalog Service for the Web (CSW)](https://en.wikipedia.org/wiki/Catalog_Service_for_the_Web), parse resulting records to obtain web data service endpoints, and retrieve data from these service endpoints. "
    ]
   },
   {
    "cell_type": "code",
    "execution_count": 1,
    "metadata": {
-    "collapsed": true,
-    "deletable": true,
-    "editable": true
+    "ExecuteTime": {
+     "end_time": "2017-04-21T09:51:58.390477",
+     "start_time": "2017-04-21T09:51:58.384477"
```

Comment:
Looks like the use of the extension is polluting the notebook. I am afraid that may get into the final HTML for the page...